formatTest/unit_tests/expected_output/wrappingTest.re

```diff
@@ -2527,15 +2530,14 @@ let letsPutAWhereClauseOnTheLast (x) =
   };
 
 type wrappingGadt(_) =
-  | ThisIsLongSoTypeWillWrap
-      (int) : wrappingGadt(int)
+  | ThisIsLongSoTypeWillWrap(int): wrappingGadt(
+                                     int
+                                   )
   | Add: wrappingGadt(((int, int) => int))
-  | App
-      (
-        wrappingGadt((('b) => 'a)),
-        wrappingGadt('b)
-      )
-      : wrappingGadt('a);
+  | App(
+         wrappingGadt((('b) => 'a)),
```

Comment:
Same comment for additional parenthesis here. It could be one of the following:
```
       wrappingGadt('b => 'a)
```
or
```
       wrappingGadt(('b) => 'a))
```